x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json

```diff
@@ -38,34 +38,16 @@
         "source.ip": "192.168.1.146",
         "source.packets": 4,
         "source.port": 32858,
-        "suricata.eve.alert.action": "allowed",
         "suricata.eve.alert.category": "Attempted Information Leak",
         "suricata.eve.alert.gid": 1,
         "suricata.eve.alert.rev": 4,
-        "suricata.eve.alert.severity": 2,
         "suricata.eve.alert.signature": "ET POLICY curl User-Agent Outbound",
         "suricata.eve.alert.signature_id": 2013028,
-        "suricata.eve.app_proto": "http",
-        "suricata.eve.dest_ip": "93.184.216.34",
-        "suricata.eve.dest_port": 80,
-        "suricata.eve.event_type": "alert",
-        "suricata.eve.flow.bytes_toclient": 1654,
-        "suricata.eve.flow.bytes_toserver": 347,
-        "suricata.eve.flow.pkts_toclient": 3,
-        "suricata.eve.flow.pkts_toserver": 4,
         "suricata.eve.flow_id": 2191386088856669,
-        "suricata.eve.http.hostname": "example.net",
         "suricata.eve.http.http_content_type": "text/html",
```

Comment:
@webmat This and the one below could be something for ECS in the future.

Reply:
Yeah I took a note to add `content_type`, when reviewing this PR, indeed :-)

What's the other field you're referring to? The field below is `method` LOL. And `.hostname` => `.domain` ;-)